src/main/java/xyz/aadev/generitech/common/blocks/power/BlockCables.java

```diff
@@ -1,6 +1,13 @@
 package xyz.aadev.generitech.common.blocks.power;
 
 
+import ibxm.Player;
+import net.minecraft.tileentity.TileEntity;
+import xyz.aadev.aalib.common.gui.GuiHandler;
+import xyz.aadev.aalib.common.logging.Logger;
+import xyz.aadev.generitech.GeneriTech;
+import xyz.aadev.generitech.Reference;
+import xyz.aadev.generitech.common.items.Items;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'xyz.aadev.generitech.common.items.Items'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AUselessImportCheck)